ci-operator/step-registry/mco/conf/day2/add-mcoqe-robot-to-pull-secret/README.md

```diff
@@ -0,0 +1,43 @@
+# add-mcoqe-robot-to-pull-secret-ref<!-- omit from toc -->
+
+## Table of Contents<!-- omit from toc -->
+- [Purpose](#purpose)
+- [Process](#process)
+- [Requirements](#requirements)
+  - [Infrastructure](#infrastructure)
+  - [Environment Variables](#environment-variables)
+
+## Purpose
+
+To add a "mcoqe" robot account's credentials to the cluster's pull-secret. These creadentials are needed to run Layering MCO test cases, since the layered osImage will be stored in the "mcoqe/layering" quay.io repository.
```

Comment:
```suggestion
To add a "mcoqe" robot account's credentials to the cluster's pull-secret. These creadentials are needed to run MCO OS Layering test cases, since the layered osImage will be stored in the "mcoqe/layering" quay.io repository.
```